x/auth/types/params.go

```diff
@@ -24,20 +25,22 @@ var (
 	KeyTxSizeCostPerByte      = []byte("TxSizeCostPerByte")
 	KeySigVerifyCostED25519   = []byte("SigVerifyCostED25519")
 	KeySigVerifyCostSecp256k1 = []byte("SigVerifyCostSecp256k1")
+	KeyPubKeyChangeCost       = []byte("PubKeyChangeCost")
```

Comment:
Should this param maybe be moved into the changepubkey module directly?  That way the module is standalone?  I'm not sure.  What do you think?

Reply:
I guess as a larger question, should this module just be moved completely out of the auth module altogether?